ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,15 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+<VirtualHost *>
```

Comment:
Can you direct me to something (docs, pastebin of some code/psuedo code, something else) that I could do to test it?